spec/requests/draft_controller_spec.rb

```diff
@@ -20,4 +20,21 @@
     expect(response.status).to eq(200)
     expect(Draft.get(user, 'xxx', 0)).to eq(nil)
   end
+
+end
+
+describe DraftsController do
```

Comment:
This has been declared above in the file. New assertions should go under a new `describe` block

```
describe "#drafts" do
  it "..." do
    assertions
  end

  it "..." do
    assertions
  end
end
```

Reply:
What's described above is `DraftController` singular, this is the plural. But fair enough, it should be renamed to `#drafts` to avoid exactly this kind of confusion.

Reply:
O wait is that a different controller or is it just a different action? If it is a different controller it should go into a separate spec file.

Reply:
It's a different controller, I'll split it out.